tests/plugin-install-order/BUILD.bazel

```
@@ -0,0 +1,90 @@
+"""
+Tests that GHC plugins are applied in the order declared in the BUILD file.
+
+Note [Plugin order]
+~~~~~~~~~~~~~~~~~~~
+
+We reverse the (counter-intuitive) order used by GHC. This test is
```

IIUC with this change we use the same order as GHC which is the counter-intuitive choice of applying the plugins in reverse order than defined in the attribute.
```suggestion
We use the same (counter-intuitive) order of plugins as used by GHC. This test is
```

---

No, the GHC order is used in master. This patch reverses the GHC order and adds a test which will break when the order is changed upstream (so that we can revert this).

---

The TODO comments they were just redundant `ghc_plugin` options. I've removed them.

---

> No, the GHC order is used in master. This patch reverses the GHC order and adds a test which will break when the order is changed upstream (so that we can revert this).

Oh, I see, I misread the issue before. Thanks for clarifying!



> The TODO comments they were just redundant `ghc_plugin` options. I've removed them.

Thanks!